addOns/help/src/main/javahelp/contents/ui/tabs/history.html

```diff
@@ -115,6 +115,11 @@ <H3>Manage History Tags...</H3>
 the <a href="../../start/features/tags.html">tags</a> associated with the
 request.
 
+<H3>Jump to History Id...</H3>
+This will bring up a dialog prompting for the Id (number) of a history entry you wish to jump to. If the entered Id value
+is not a visible history item then nothing will happen.
+Also accessible via CTRL + ALT + J.
```

Comment:
"accessible by default via".. ?
Could also link to https://www.zaproxy.org/docs/desktop/ui/dialogs/options/keyboard/ ?

Reply:
I'll have to look at plugging it in to the keyboard options handling. Currently it's static.

Reply:
Actually I don't think we currently have a way to make this cusomizable.

Reply:
Oh :( Do we need core changes to allow that?

Reply:
I think so.

I'll keep looking into it but I'd rather it not hold this up.

Reply:
I approved it anyway - core changes would be a new PR in any case 😁

Reply:
Thanks